lib/src/components/NxDivider/NxDivider.tsx

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import withClass from '../../util/withClass';
+import './NxDivider.scss';
+
+export const NxDivider = withClass('hr', 'nx-divider');
```

Comment:
Needs to be deleted

Reply:
> Needs to be deleted

Check e1f79dfe5abbe6861ef08f7b224747b2b200e2dd